src/uniform.rs

```diff
@@ -1,3 +1,5 @@
+/// # Safety
```

Comment:
Move this block to the end of doc comment and add this
"The type must not have any padding bytes"

Reply:
Done! Thanks